### ISSUE TITLE
[BUG] when parsing --dimensions, $5 value is not verified.

### ISSUE THREAD

**Issue by Anvil** [aristocratos/bashtop#30]
I have not run the program, i was just reading the source code.

I *think* you made a booboo:

```bash
-d|-dimensions) if [[ $2 =~ ^[0-9]+$ && $3 =~ ^[0-9]+$ && $3 =~ ^[0-9]+$ && $4 =~ ^[0-9]+$ ]]; then 				#? Graph dimensions
		line=$2; col=$3; height=$4; width=$5; shift 4; fi;;
```
You're testing $3 value twice, and you're not testing $5.

Also, since you're using it a lot, you could actually refactor that test a function, it would improve readability:

```bash
_all_int() {
    local param
    for param; do
        [[ $param =~ ^[0-9]+$ ]] || return
    done
}
```

For --dimensions parsing, you can call it like this:
```bash
if _all_int "${@:2:4}"; then line=$2 ... ; shift 4 ; fi ;;
```

**Comment by aristocratos** [comment]
I sure did do a booboo :)
Luckily it doesn't break anything.
And thanks for the idea about a int check function!

**Comment by aristocratos** [comment]
Fixed in v0.8.9
But I'm sure there are more mishaps like that, a lot of  the code was written under various states of sleep deprevation.
